js/start.js

```diff
@@ -39,8 +39,20 @@ app.localSettings.fetch().fail(() => app.localSettings.save());
 const pageNav = new PageNav();
 $('#pageNavContainer').append(pageNav.render().el);
```

Comment:
If we find we're targeting `#pageNavContainer` or `#chatContainer` anywhere else, then we should add it to "selectors" module.